bluesky/simple_scans.py

```diff
@@ -66,8 +67,8 @@ class _BundledScan:
 
     def __init__(self):
         # subs and sub_factories can be set individually per instance
-        self.subs = self.default_subs
-        self.sub_factories = self.default_sub_factories
+        self.subs = list(self.default_subs)
```

Comment:
There are internal data structures to convenience classes, if we are going to do DWIM here, then it needs to happen a layer up and be stored in the attributes normalized.

If this is supposed to be user facing in this case it is even worse as now we are exposing either, a dict _or_ a list to the user, how are they supposed to know what method to use to add to it?